src/session/content/message_row/bubble.rs

```diff
@@ -218,7 +218,8 @@ impl MessageBubble {
                 sender_name_expression.bind(&*imp.sender_label, "label", glib::Object::NONE);
             imp.sender_binding.replace(Some(sender_binding));
 
-            self.update_sender_color(maybe_id);
+            //Color sender name only if message is incoming
+            self.update_sender_color(maybe_id, !message.is_outgoing());
```

Comment:
Mh, why not just setting `show_sender` to `None` when the message is outgoing instead of doing this?

Reply:
The message is indeed outgoing, but that doesn't mean you're the sender of the message. E.g. you wrote a post in the channel and it's visible as an outgoing message in the discussion group, in that case we want to show sender name even though the message is outgoing.

Reply:
Ah wait, I meant to set the first parameter of `update_sender_color` (`maybe_id`) to `None` when the message is outgoing.